aws/resource_aws_redshift_snapshot_copy_grant.go

```diff
@@ -260,6 +275,9 @@ func findAwsRedshiftSnapshotCopyGrant(conn *redshift.Redshift, grantName string,
 
 		return nil
 	})
+	if isResourceTimeoutError(err) {
```

Comment:
It looks like the `resource.Retry()` above here may be extraneous as it has no `return resource.RetryableError()` 👍